EnoCore/Models/Json/JsonConfigurationService.cs

```diff
@@ -9,9 +10,24 @@ public class JsonConfigurationService
 #pragma warning disable CS8618
         public long Id { get; set; }
         public string Name { get; set; }
-        public long FlagsPerRound { get; set; }
-        public long NoisesPerRound { get; set; }
-        public long HavocsPerRound { get; set; }
+        [JsonPropertyName("FlagsPerRound")]

```

Comment:
This is just a multiplication factor, right? Can we rename it to something that reflects that change?

Reply:
Do you refer to the name of the config option in Json or to the name of the variable? I did not want to change the code too much for this, this is why I choose the option with the additional property.

Reply:
Both, but the config option is more urgent, since the name does no longer correlate with what it is used for